index.html

```diff
@@ -0,0 +1,245 @@
+                <td class="t-title">Lecture</td>
+                <td class="t-description">Listen to the speakers from various
+                  countries about
+                  they cientific studies related to evolution.</td>
+              </tr>
+            </tbody>
+          </table>
+
+          <table class="table table-borderless t-main">
+            <tbody class="main-column">
+              <tr class="main-row">
+                <td class="t-img"><img class="main-img" src="Images/main_02.png"
+                    alt="Exhibit Icon"></td>
+                <td class="t-title">Exhibition</td>
+                <td class="t-description">Appreciate several fossils and their
+                  age that show
+                  that evolution takes millions of years.</td>
+              </tr>
+            </tbody>
+          </table>
+
+          <table class="table table-borderless t-main">
+            <tbody>
+              <tr class="main-row">
+                <td class="t-img"><img class="main-img" src="Images/main_03.png"
+                    alt="Forum Icon"></td>
+                <td class="t-title">Forum</td>
+                <td class="t-description">Have the time to share your thoughts
+                  and opinions with
+                  experts for each topic.</td>
+              </tr>
+            </tbody>
+          </table>
+
+          <table class="table table-borderless t-main">
+            <tbody>
+              <tr class="main-row">
+                <td class="t-img"><img class="main-img" src="Images/main_04.png"
+                    alt="Workshop Icon"></td>
+                <td class="t-title">Workshop</td>
+                <td class="t-description">Can the science prove that everything
+                  was created from nothing,
+                  come and discover it yourself.</td>
+              </tr>
+            </tbody>
+          </table>
+
+          <table class="table table-borderless t-main">
+            <tbody>
+              <tr class="main-row">
+                <td class="t-img"><img class="main-img" src="Images/main_05.png"
+                    alt="Networking Icon"></td>
+                <td class="t-title">Ignite</td>
+                <td class="t-description">Get opportunities to network with
+                  people from all over the world
+                  trying to know the truth.</td>
+              </tr>
+            </tbody>
+          </table>
+
+        </div>
+
+        <div class="joinButton" id="join">
+          <button class="join">
+            Join CvsE Global Summit 2023
+          </button>
+        </div>
+
+        <div class="program">
+          <a class="see-program" href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+
+      <section class="speakers" id="speakers">
+        <div class="featured">
+          <h4>Featured Speakers</h4>
+        </div>
+        <div class="main-bar"></div>
+
+        <div class="grid-container">
+
```

Comment:
- Kindly add the for `hover` state white border to improve the user experience 👍
![2023-04-20_23-11](https://user-images.githubusercontent.com/52010789/233477630-8f2ad8bc-d4e1-430d-8615-1a5b832e2e3e.png)

Reply:
- [ ] Again here as requested by the previous reviewer, kindly add a hover effect of a white border around the articles to improve user experience. I think the previous reviewer meant a `hover effect` as is shown in the given design

Given design
![image](https://user-images.githubusercontent.com/81932478/233687967-fb701f23-e991-49bd-afe9-6fd78e015e82.png)

#### Note that the fourth article in the screenshot above has a white border showing that the cursor is hovering over it. 

Your design
![image](https://user-images.githubusercontent.com/81932478/233688330-500168e2-ced3-4168-8fc6-0b0db84718dd.png)